tools/azure-sdk-tools/devtools_testutils/proxy_startup.py

```diff
@@ -148,7 +171,7 @@ def start_test_proxy() -> None:
     set_custom_default_matcher(excluded_headers=headers_to_ignore)
 
 
-def stop_test_proxy() -> None:
+def stop_test_proxy(request) -> None:
```

Comment:
Doesn't hurt to accept `request`, but it looks like we can also drop it for now

Reply:
Right! Cleaning it. thanks for the catch